tensorflow_blade/build.py

```diff
@@ -272,7 +285,10 @@ def _write(line, cmd="build"):
 
 def build_with_bazel(args):
     with cwd(ROOT):
-        execute("bazel build //src:_tf_blade.so")
+        if args.device == "cpu":
+            execute("bazel build @org_tao_bridge//:libtao_ops.so")
```

Comment:
why not build tf_blade.so for all?

Reply:
Currently `tf_blade` has no cpu related code ..., just a placeholder here.

Reply:
Modified to build `//src:_tf_blade.so` with different configs